tests/test_userstory_39.py

```diff
@@ -56,3 +62,87 @@ def test_melding_overlast():
         'object': verblijfsobject['url'],
     })
     assert 'url' in zaak_object
+
+
+    '''
+    Create temporary files to test file uploading
+    '''
+    # Text file (can't be empty or the API throws an error)
+    fd, txt_path = tempfile.mkstemp()
+    with open(txt_path, 'w') as tmp:
+        tmp.write('some additional information')
+
+    # Image (png format)
+    im_format = 'png'
```

Comment:
look into pytest fixtures to get 'some file' - advantage is that you can add cleanup hooks etc. For an example, you can check https://github.com/sergei-maertens/django-privates/blob/develop/tests/conftest.py#L9 as a reference

Reply:
used a conftest.py file